servicetalk-capacity-limiter-api/src/main/java/io/servicetalk/apple/capacity/limiter/api/Preconditions.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.apple.capacity.limiter.api;
+
+import java.time.Duration;
+
+import static java.time.Duration.ZERO;
+
+final class Preconditions {
+
+    private Preconditions() {
```

Comment:
How many of these can instead use `NumberUtils`?

Reply:
👍  consolidated there.